doc/Install.md

```diff
@@ -26,12 +26,12 @@ Requirements
 ---
 
 * Apache with mod-rewrite enabled and "Options All" so you can use a local .htaccess file
-* PHP  5.2+. The later the better. You'll need 5.3 for encryption of key exchange conversations. On a Windows environment, 5.2+ might not work as the function dns_get_record() is only available with version 5.3.
+* PHP 5.5+.
```

Comment:
I couldn't remember the post so I just guessed :-) I can set it to 5.4, there is no need for the mysql stuff for higher versions.